storm/templates/storm.yaml.j2

```diff
@@ -43,7 +43,7 @@ nimbus.seeds:
 nimbus.thrift.port: {{ NIMBUS_THRIFT_PORT | default('6627') }}
 nimbus.childopts: "{{ NIMBUS_CHILDOPTS }}"
 
-ui.host: "{{ STORM_UI_HOST | default ('127.0.0.1') }}"
+ui.host: 0.0.0.0
```

Comment:
I would prefer to allow operator to change `ui.host` if he want to, so instead of removing all configuration option just replace default value: `"{{ STORM_UI_HOST | default ('0.0.0.0') }}"`